percona/config.sls

```diff
@@ -27,23 +23,18 @@ mysql_python_dep:
 {%     else %}
 {%       set filepath = percona_settings.config_directory + '/' + file %}
 {%     endif %}
-{{ filepath }}:
-  file.managed:
-    - user: root
-    - group: root
-    - mode: 0644
-    - source: salt://percona/files/mysql.cnf.j2
-    - template: jinja
-    - context:
-        config: {{ content |default({}) }}
-    - require_in:
-      - pkg: percona_client
-      - pkg: percona_server
-    - require:
-      - file: {{ percona_settings.config_directory }}
+{%     if not extend_written %}
+extend:
```

Comment:
Instead of using extend, we should check if `percona.config` has content.

Reply:
Done in:  
b02e19bec22865e60a2d85dd82dece512af036d4